### PR TITLE
Create usage documentation

### PR DESCRIPTION
**Pull request by buchholz-sean**
Documentation for Max Bot usage once installed. Includes a link in README.md to the usage doc.

### CODE REVIEW

File: usage.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Adds all available Armada roles to the user. Suitable for masochists and chat room junkies.
+
+### !removeAllRoles
+
+Example: `!removeAllRoles`
+
+Removes all roles from the user. _Tabula rasa_, baby.
+
+
+## Administrative commands
```
Comment (buchholz-sean) [inline usage.md:58]: Note that the broadcast functionality hasn't been added to a release yet. Everything from line 58 down can be scrapped or edited as necessary.

**Review by MROwens — APPROVED** [review]
Everything looks good to me!

**Comment by ghost** [comment]
would look into getting some documentation for nodemailer seeing as we are using there services to setup the email verification, possible option could be:

## nodemailer

```npm install nodemailer --save```
Max-Bot uses [nodemailer](https://nodemailer.com/about/) to send verification emails to students that are attempting to verify their Full Sail University email address.

**Comment by buchholz-sean** [comment]
Great call, @joegreen2. Will update.

**Review by ghost — APPROVED** [review]
LGTM :+1:

**Pull request merged**